src/OrbitGl/include/OrbitGl/CaptureViewElement.h

```diff
@@ -164,13 +164,24 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
   [[nodiscard]] virtual EventResult OnMouseEnter();
   [[nodiscard]] virtual EventResult OnMouseLeave();
 
+  [[nodiscard]] virtual bool RestrictDrawingToBody() const { return false; }
+
+  [[nodiscard]] uint32_t GetUid() const { return uid_; }
+
  private:
   bool is_mouse_over_ = false;
+  uint32_t uid_;
```

Comment:
Should this be initialized?

Reply:
Also this should not be in this PR. Actually all changes in `CaptureViewElement.h` should not be included here, sorry for that.